timer/api/timer.go

```diff
@@ -64,6 +64,43 @@ func (p *SchedIntervalPolicy) NextEventTime(watermark time.Time) (time.Time, boo
 	return watermark.Add(p.interval), true
 }
 
+// ManualRequest is the request info to trigger timer manually
```

Comment:
```suggestion
// ManualRequest is the request info to trigger timer manually.
```
Ditto for the other comments.